### PR TITLE
Added 8 new snippets and made folders for some snippets for convenience.

### PR DESCRIPTION
**Pull request by skadimoolam**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .DS_Store
 *.cache
-*.pyc
\ No newline at end of file
+*.pyc
+test.txt
```
Comment (ybakos) [inline .gitignore]: test.txt and test.pde don't belong in the gitignore.

File: Snippets/Date and Time/day.sublime-snippet [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<snippet>
+    <content><![CDATA[day();]]></content>
+    <description>day</description>
+    <scope>source.pde</scope>
+    <tabTrigger>time</tabTrigger>
```
Comment (ybakos) [inline Snippets/Date and Time/day.sublime-snippet]: The `tabTrigger` of `time` seems a little counter-intuitive. How about `day` ?

Comment (ybakos) [inline Snippets/Date and Time/day.sublime-snippet]: Wait, I see, the others use time as well. Ok, keep this as-is to be consistent.

File: Snippets/Mouse/mouseClicked.sublime-snippet [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<snippet>
+    <content><![CDATA[
+    	mouseClicked(${1:event}){
```
Comment (ybakos) [inline Snippets/Mouse/mouseClicked.sublime-snippet]: Missing return type, and see the other snippets for formatting (need a space between closing parens and opening brace.

Comment (ybakos) [inline Snippets/Mouse/mouseClicked.sublime-snippet]: Should also be outdented, like you did with `mouseWheel`.

File: Snippets/Mouse/mouseWheel.sublime-snippet [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<snippet>
+    <content><![CDATA[
+void mouseWheel(${1:MouseEvent event}){
```
Comment (ybakos) [inline Snippets/Mouse/mouseWheel.sublime-snippet]: Need a space between closing parens and opening brace.

File: Snippets/Typography/createFont_1.sublime-snippet [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<snippet>
+    <content><![CDATA[PFont ${1:myFont} = createFont(${2:name}, ${3:size});]]></content>
+    <description>createFont(name,size)</description>
```
Comment (ybakos) [inline Snippets/Typography/createFont_1.sublime-snippet]: Semicolon?

Comment (ybakos) [inline Snippets/Typography/createFont_1.sublime-snippet]: And, use a space after commas in the argument list. `createFont(name, size);`.

**Comment by ybakos** [comment]
Thanks @skadimoolam ! Just see my comments.

**Comment by skadimoolam** [comment]
I'll make the necessary changes, Thanks.

**Pull request closed without merging**